utils/misc/dbutils.go

```diff
@@ -3,32 +3,51 @@ package misc
 import (
 	"database/sql"
 	"fmt"
+	"os"
 	"strconv"
+	"sync"
 
 	"github.com/lib/pq"
 
 	"github.com/rudderlabs/rudder-server/config"
 )
 
 var (
-	host, user, password, sslmode string
-	port                          int
+	host, user, password, dbname, sslmode, appName string
+	port                                           int
+	once                                           sync.Once
 )
 
 func loadConfig() {
 	host = config.GetEnv("JOBS_DB_HOST", "localhost")
 	user = config.GetEnv("JOBS_DB_USER", "ubuntu")
+	dbname = config.GetEnv("JOBS_DB_DB_NAME", "ubuntu")
 	port, _ = strconv.Atoi(config.GetEnv("JOBS_DB_PORT", "5432"))
 	password = config.GetEnv("JOBS_DB_PASSWORD", "ubuntu") // Reading secrets from
 	sslmode = config.GetEnv("JOBS_DB_SSL_MODE", "disable")
+	// Application Name can be any string of less than NAMEDATALEN characters (64 characters in a standard PostgreSQL build).
+	// There is no need to truncate the string on our own though since PostgreSQL auto-truncates this identifier and issues a relevant notice if necessary.
+	appName = DefaultString("rudder-server").OnError(os.Hostname())
+}
+
+// GetConnectionString Returns Jobs DB connection configuration
+func GetConnectionString() string {
+	once.Do(func() {
+		loadConfig()
+	})
```

Comment:
The usage of once here might make testing harder, depending on how this function is used. What was the intention of its usage here?